app/herdbook.py

```diff
@@ -238,33 +241,82 @@ def individual(i_id):
     """
     user_id = session.get("user_id", None)
     ind = da.get_individual(i_id, user_id)
+
     if ind:
-        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['id']) * 100)
+        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['id'], ind['genebank_id']) * 100)
     return jsonify(ind)
 
-def get_inbreeding(i_id):
+
+def get_inbreeding(i_id, g_id):
     """
     Returns  the inbreeding coefficient of the individual given by `i_id`.
     """
     id = str(i_id)
-    coefficients = load_inbreeding()
+    coefficients = load_inbreeding(g_id)
     if id in coefficients:
         return coefficients[id]
     return None
 
 
-@APP.route("/api/inbreeding/")
-def all_inbreeding():
-    coefficients = load_inbreeding()
-    return jsonify({"coefficients": coefficients})
+@APP.route("/api/<int:g_id>/inbreeding/")
+def inbreeding(g_id):
+    id = str(g_id)
+    inb_coeffcient = load_inbreeding(id)
+    return jsonify(inb_coeffcient)
+
+
+def load_inbreeding(g_id):
+    response = requests.get('http://{}:{}/inbreeding/{}'.format(settings.rapi.host, settings.rapi.port, g_id),
+                             params={})
+
+    if response.status_code == 200:
+        return csvparser.parse_csv(response)
```

Comment:
```suggestion
        return csvparser.parse_csv(response.content)
```
Just send the response content instead of the whole response object?

Reply:
Yes, good idea, since we are not using the latter for anything else.